### DISCUSSION TITLE
Authentication Fails

### DISCUSSION THREAD

**Discussion by davpel** [samschott/maestral#478] [Q&A]
Just installed Maestral on an M1 MacBook Mini running the latest version of MacOS. Every time I try to link the app with Dropbox, it seems to accept the code but then when it starts to sync, I get "Authentication Failed. Path: rev:10d3dc79bf4
Please make sure that you entered the correct authentication code."

**Comment by samschott** [comment]
I've seen this as well just now. Possibly just a temporary glitch from Dropbox servers. Does it persist after restarting Maestral?

**Reply by JordanSavoie** [reply]
I have the same problem on an M1 Mac Mini. Starts fine and syncs any changes as I make them. When I press "resume sync" it works fine, syncs any changes normally, and doesn't complain for a while until the same message comes up again.

I tried restarting and it doesn't help.

**Comment by davpel** [comment]
Yes, the problem persists when I restart Maestral.

**Comment by brandoseven** [comment]
Same here. On a Mac Mini M1 and a Macbook Air M1.

**Comment by JordanSavoie** [comment]
Same problem also happens on my MacBook Pro running Catalina

**Comment by samschott** [comment]
I think this should be fixed now, though I haven't gotten a confirmation yet from the Dropbox team.
